### PR TITLE
refactor(react-ui-kit): Refactor file structure

### PR DESCRIPTION
**Pull request by AndyTheGiant**

### CODE REVIEW

File: packages/react-ui-kit/src/Form/Button.jsx [hunk 1]
```diff
@@ -17,10 +17,10 @@
  *
  */
 
-import {COLOR} from './variables';
+import {COLOR} from '../variables';
```
Comment (bennyn) [inline packages/react-ui-kit/src/Form/Button.jsx]: `import {COLOR} from '../variables';` -> `import {COLOR} from '../Theme';` or sth. similar

File: packages/react-ui-kit/src/Grid/Column.jsx [hunk 2]
```diff
@@ -17,8 +17,8 @@
  *
  */
 
-import {SIZE} from './variables';
-import {media} from './mixins';
+import {SIZE} from '../variables';
```
Comment (bennyn) [inline packages/react-ui-kit/src/Grid/Column.jsx]: Sizes can be part of the "Grid"

File: packages/react-ui-kit/package.json [hunk 3]
```diff
@@ -35,5 +23,24 @@
...
+    "clear": "rimraf dist",
+    "dist": "yarn clear && yarn babel src --out-dir dist --no-comments",
+    "fix:script": "yarn test --fix",
+    "fix": "yarn fix:script",
+    "precommit": "lint-staged",
+    "prepare": "yarn dist && node scripts/prepare.js",
+    "start": "yarn start:demo",
+    "start:static": "gulp",
+    "start:demo": "webpack-dev-server",
+    "test": "eslint -c .eslintrc.yaml --ignore-path .gitignore --ignore-path .prettierignore '**/*.{js,jsx}'",
```
Comment (lipis) [inline packages/react-ui-kit/package.json]: remove the test from here and the fix related stuff

Comment (lipis) [reply packages/react-ui-kit/package.json]: precommit etc.. the ones that are red in this diff :D

File: packages/react-ui-kit/src/Identity/colors.js [hunk 4]
```diff
@@ -106,18 +106,3 @@ export const COLOR = {
   ...LIGHT_COLOR,
   ...OPAQUE_COLOR,
 };
-
-export const FONT = {
-  fontFamily: '-apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica Neue, Helvetica, Arial, sans-serif',
-};
```
Comment (lipis) [inline packages/react-ui-kit/src/Identity/colors.js]: I would argue that we could keep that even if it's used only in one place.. just to be able to find it most likely inside the variables/settings section..

Comment (AndyTheGiant) [reply packages/react-ui-kit/src/Identity/colors.js]: nope, we don't need it.

Comment (lipis) [reply packages/react-ui-kit/src/Identity/colors.js]: :(

**Pull request merged**